### PR TITLE
Initial winter_peak.py readme

### PR DESCRIPTION
**Pull request by mdallaire**
Initial readme to try and standardise on the various winter credit related terms and their meanings and usages.

The idea is to get a discussion going on a naming scheme that is closely related to the terms used by HQ while still making sense in the context of an home automation system and it's logics.

### CODE REVIEW

**Comment by mdallaire** [comment]
There are some terms that changed after reading HQ's documentation, temperature adjustment vs reference is a big change I think but we should probably change it in the module to better reflect HQ's terms.

File: winter_credit/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+### peak (peak hours)
+
+In this module: All hours from 06:00 to 09:00 and from 16:00 to 20:00 during the winter.
+
+This is when the critical peak events from hydro are happening. 
+
+In hydro's document there are also exclusions for specific holiday dates (Christmans, New year, Good Friday and Easter Monday) that we don't take into account here (yet)
+
+### temperature adjustement
```
Comment (mdallaire) [inline winter_credit/README.md:30]: Anciennement période de référence.

Ce [document](http://publicsde.regie-energie.qc.ca/projets/469/DocPrj/R-4057-2018-B-0062-DDR-RepDDR-2018_10_26.pdf#page=125) de la régie de l'énergie fait aussi référence à une période d'ancrage pour nommé cette période.

File: winter_credit/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+> I am unsure on how to name this. I thought reference made sense but in HQ's doc it means something completely different (see below)
+
+This period starts 5 hours before the next peak event's start time and has a duration of 3 hours. With the current peak period (as described above) it results in the following time periods:
```
Comment (mdallaire) [inline winter_credit/README.md]: Anciennement période de référence.

Ce [document](publicsde.regie-energie.qc.ca/projets/469/DocPrj/R-4057-2018-B-0062-DDR-RepDDR-2018_10_26.pdf#page=125) de la régie de l'énergie fait aussi référence à une période d'ancrage pour nommé cette période.

Comment (mdallaire) [reply winter_credit/README.md]: I switched temperature adjustement to anchor period. It is a simpler term and probably less misleading than temperature adjustement that could be confused with the pre-heating period.

**Comment by zepiaf** [comment]
Sorry for the delay

Do you mind if we push that in the documentation ?

I think it's good but I'd rather avoid managing two sets of doc :) I can put a converted version of this in rst format and have it in the readthedoc thingy

**Comment by mdallaire** [comment]
> Sorry for the delay
> 
> Do you mind if we push that in the documentation ?
> 
> I think it's good but I'd rather avoid managing two sets of doc :) I can put a converted version of this in rst format and have it in the readthedoc thingy

Done!

**Comment by clauderobi** [comment]
This is good. Everything seems there. 

I would add some more clarification text similar to: To be sure, peak periods may be normal or critical and exist everyday.

**Comment by mdallaire** [comment]
Integrated in #22

**Pull request closed without merging**